### PR TITLE
RK-210 Build Fraud Value Page

### PR DESCRIPTION
**Pull request by shiromi**

### CODE REVIEW

File: test/controllers/ApproximateValueControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        val boundForm = form.bind(Map("value" -> "invalid value"))
+
+        val view = application.injector.instanceOf[ApproximateValueView]
+
+        val result = route(application, request).value
+
+        status(result) mustEqual BAD_REQUEST
+        contentAsString(result) mustEqual view(boundForm, NormalMode)(request, messages(application)).toString
+      }
+    }
```
Comment (shinealex33) [inline test/controllers/ApproximateValueControllerSpec.scala:130]: We need to have test cases for three negative scenarios- please test case for below
1. provide empty value
2. whole Number
3. Non numeric

**Review by shinealex33 — CHANGES_REQUESTED** [review]
Please add missing test cases. Thank you!

**Review by shinealex33 — APPROVED** [review]
LGTM
